### ISSUE TITLE
Null timestamps for meeting_item: a typo in the type field names?

### ISSUE THREAD

**Issue by anna-kartynnik** [codeforsanjose/gov-agenda-notifier#151]
I get item's timestamps as `null` even if they are saved in the db:

```
        {
          "id": 17,
          "meeting_id": 10,
          "parent_meeting_item_id": null,
          "order_number": 1,
          "status": "PENDING",
          "meeting_start_timestamp": null,
          "meeting_end_timestamp": null,
          "content_categories": "test"
        }
```
<img width="1101" alt="Screen Shot 2021-03-09 at 9 00 28 PM" src="https://user-images.githubusercontent.com/15210801/110578876-8545a980-811a-11eb-91c0-361ad9f850b2.png">

I think this is because we have a typo in the `meeting_item` type here:
https://github.com/codeforsanjose/gov-agenda-notifier/blob/1a3ece256fcf0361c21e6106cefa1b58b14ab49e/backend/graphql_api/lambda/graphql/apolloServer.js#L82-L95

Here we have `meeting_start_timestamp` and `meeting_end_timestamp` fields whereas `meeting_item` entity has fields `item_start_timestamp` and `item_end_timestamp` in the db (and everywhere else in the code). If anyone can confirm that it is a typo, I can create a PR.

**Comment by sent-hil** [comment]
@anna-kartynnik yea looks like a typo

**Comment by anna-kartynnik** [comment]
Yep, it does. I'll create a PR.

**Comment by anna-kartynnik** [comment]
Fixed in PR#160.
